wsmaster/che-core-api-devfile/src/main/resources/schema/devfile.json

```diff
@@ -155,6 +189,128 @@
             "examples": [
               "{\n   \"app.kubernetes.io/name\" : \"mysql\", \n   \"app.kubernetes.io/component\" : \"database\", \n   \"app.kubernetes.io/part-of\" : \"petclinic\" \n}"
             ]
+          },
+          "image": {
+            "type": "string",
+            "description": "Described the docker image that should be used for tool",
```

Comment:
`Describes`  (or, even specifies, i would say, for exact values :) )

Reply:
Thanks for catching this copy/paste error

Reply:
Now it's 
```json
"description": "Specifies the docker image that should be used for tool"
```